sim/druid/mangle.go

```diff
@@ -35,22 +45,33 @@ func (druid *Druid) registerMangleBearSpell() {
 			IgnoreHaste: true,
 			CD: core.Cooldown{
 				Timer:    druid.NewTimer(),
-				Duration: time.Duration(float64(time.Second) * (6 - durReduction)),
+				Duration: time.Second * 6,
 			},
 		},
+		// TODO: Berserk 3 target mangle cleave - Saeyon
 
-		DamageMultiplier: (1 + 0.1*float64(druid.Talents.SavageFury)) * 1.15
-		ThreatMultiplier: core.TernaryFloat64(druid.HasSetBonus(ItemSetThunderheartHarness, 2), 1.15, 1),
+		DamageMultiplier: 1.6 * (1 + 0.1*float64(druid.Talents.SavageFury)) * idolMultiplier,
```

Comment:
These are mixing the additive and multiplicative multipliers so we should modify this a bit
```suggestion
		DamageMultiplier: 1.6 * idolMultiplier,
		DamageMultiplierAdditive: 1 + 0.1*float64(druid.Talents.SavageFury)
```